lib/components/layout/ModalLayout.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { PropsWithChildren } from 'react';
+import { Box, Divider, Stack, Typography } from '@mui/material';
+import vars from '../assets/styles/variables';
+
+interface IMainProps {
+    title: string;
+    headerLeftNode?: React.ReactNode;
+    footerNode?: React.ReactNode;
+    footerTopElement?: React.ReactNode;
+}
+
+const { textWhite } = vars;
+
+export const ModalLayout = ({
```

Comment:
This is the base layout for all the different views. It was taken from Emeka work. I would like to make this component even more generic so that we could use it for all the different steps we have.